hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemDelegationSAS.java

```diff
@@ -64,7 +64,7 @@
       LoggerFactory.getLogger(ITestAzureBlobFileSystemDelegationSAS.class);
 
   public ITestAzureBlobFileSystemDelegationSAS() throws Exception {
-    // These tests rely on specific settings in azure-auth-keys.xml:
+    // These tests rely on specific settings in abfs-testrun-configs.xml:
```

Comment:
settings I have in azure-auth-keys still get picked up right? This is an addition to the current test run, not a replacement?

Reply:
Basically we are renaming azure-auth-keys to abfs-testrun-configs. Because we expect the same to contain non auth related configs too.
In the main config file (azure-test.xml the xinclude is now made to abfs-testrun-configs)

Reply:
Please revert. I am not worried that it contains extra details and prefer backwards compatibility.

FWIW my auth-keys XIncludes to somewhere outside the source tree which already contains a lot more binding info than the secrets; it's managed under SCM in that private (local only) git repo and I am happy with the process. Whatever process is developed here, it MUST allow me to refer to template options in absolute paths outside of this source tree.